src/main/java/com/j256/ormlite/table/SchemaUtils.java

```diff
@@ -0,0 +1,267 @@
+package com.j256.ormlite.table;
+
+import com.j256.ormlite.dao.BaseDaoImpl;
+import com.j256.ormlite.dao.Dao;
+import com.j256.ormlite.dao.DaoManager;
+import com.j256.ormlite.db.DatabaseType;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.field.FieldType;
+import com.j256.ormlite.logger.Logger;
+import com.j256.ormlite.logger.LoggerFactory;
+import com.j256.ormlite.misc.IOUtils;
+import com.j256.ormlite.misc.SqlExceptionUtil;
+import com.j256.ormlite.stmt.StatementBuilder.StatementType;
+import com.j256.ormlite.support.CompiledStatement;
+import com.j256.ormlite.support.ConnectionSource;
+import com.j256.ormlite.support.DatabaseConnection;
+import com.j256.ormlite.support.DatabaseResults;
+
+import java.sql.SQLException;
+import java.util.*;
+
+/**
+ * Couple utility methods for the creating, dropping, and maintenance of schemas.
+ *
+ * @author graywatson
```

Comment:
I've been thinking of dropping the @author tags but at least this should be you not me.

Reply:
Removed